popstats/README.md

```diff
@@ -0,0 +1,47 @@
+# PoP stats
+
+This PR adds a service that will identify all the PoP payout transactions, summarize, calculate the incentive program points and report those to Absinthe.
```

Comment:
It's a minor thing, but, weirdly, the README starts with "This PR" - once merged it will no longer make sense... Were you thinking of a changelog or something?

Having said that, some minor corrections:

```suggestion
This PR adds a service that identifies all the PoP payout transactions, summarizes, calculates the incentive program points, and reports those to Absinthe.
```